pkgs/applications/editors/ox/default.nix

```diff
@@ -2,16 +2,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "ox";
-  version = "0.6.7";
+  version = "0.6.10";
 
   src = fetchFromGitHub {
     owner = "curlpipe";
     repo = pname;
     rev = version;
-    hash = "sha256-UFNOW/INV/65C6UysKi9lGw+PIj2NXF6ejG5UY702/I=";
+    hash = "sha256-7PaAcVatm/gqeZRuzCjoF6ZGDP6mIjDTuhmJQ5wt7x8=";
   };
 
-  cargoHash = "sha256-sy/RNMXJn5k9qw0ghCQA7PqZokpDw0xns4abwa938Gk=";
+  cargoHash = "sha256-2Jk8uDiTGUQqLOOQVlYm5R7qQXIqP0PkFvv5E5qTzT0=";
 
   buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [ darwin.apple_sdk.frameworks.AppKit ];
```

Comment:
```suggestion
```

Please switch to the new darwin sdk pattern: https://discourse.nixos.org/t/the-darwin-sdks-have-been-updated/55295

Reply:
Thank you for the suggestion, I didn't know the tread. 
Applied the changes in https://github.com/NixOS/nixpkgs/pull/354280/commits/0b94b53f034019c24d8d5345b78fdaf3a88b222b 🙏